### PR TITLE
NFC: AP_RangeFinder: Make measurement time and call back time static constant

### PR DESCRIPTION
**Pull request by khancyr**
rework from https://github.com/ArduPilot/ardupilot/pull/10092

### CODE REVIEW

**Comment by rmackay9** [comment]
This looks pretty straightforward to me actually and only affects one driver so I think this is probably safe to merge

File: libraries/AP_RangeFinder/AP_RangeFinder_VL53L1X.h [hunk 1]
```diff
@@ -1253,6 +1253,7 @@ class AP_RangeFinder_VL53L1X : public AP_RangeFinder_Backend
     // get a reading
     bool get_reading(uint16_t &reading_cm);
     AP_HAL::OwnPtr<AP_HAL::I2CDevice> dev;
+    static const uint8_t MEASUREMENT_TIME_MS = 50; // Start continuous readings at a rate of one measurement every 50 ms
```
Comment (tridge) [inline libraries/AP_RangeFinder/AP_RangeFinder_VL53L1X.h:1256]: should be in the single cpp file that uses it as a #define
header file is an interface to the outside world

**Review by tridge — CHANGES_REQUESTED** [review]

File: libraries/AP_RangeFinder/AP_RangeFinder_VL53L1X.h (hunk 1 above)
Comment (khancyr) [reply libraries/AP_RangeFinder/AP_RangeFinder_VL53L1X.h:1256]: it shouldn't matter as here we are in the class scope

**Comment by khancyr** [comment]
@tridge change done!

**Review by tridge — APPROVED** [review]

**Pull request merged**